r/R/dplyr-funcs-datetime.R

```diff
@@ -189,6 +189,70 @@ register_bindings_datetime <- function() {
   })
 }
 
+register_bindings_duration <- function() {
+  register_binding("difftime", function(time1,
+                                        time2,
+                                        tz,
+                                        units = c("auto", "secs", "mins",
+                                                  "hours", "days", "weeks")) {
+    units <- match.arg(units)
+    if (units != "secs") {
+      abort("`difftime()` with units other than seconds not supported in Arrow")
+    }
+
+    # for time32() we do not need to worry about timezone
```

Comment:
Is this stale?

Reply:
Probably. I cannot recall why it is there and also it's confusing rather than clarifying anything. Removed